$MirOS$

Revert to the non-GFDL'd man page of zsnes 1.40,
with a few adjustments.

--- src/linux/zsnes.1.orig	Tue Jan 18 00:39:51 2005
+++ src/linux/zsnes.1	Sun Oct  2 18:00:18 2005
@@ -1,173 +1,176 @@
-.TH ZSNES 1 "16 January 2005"
+.TH ZSNES 1 "26 FEB 2002"
+.SH VERSION 1.337
 
-.SH NAME
-zsnes - The ZSNES Super Nintendo emulator, version 1.41
+.SH ZSNES
+zsnes - Super Nintendo console emulator
+
 .SH SYNOPSIS
-.B zsnes
-[-?] [-0] [-1 #] [-2 #] [-7] [-8] [-9] [-a] [-cb] [-cc] [-cs] [-dd] [-f #] [-h | -l] [-i] [-j] [-k #] [-m] [-n] [-om] [-p #] [-r #] [-s] [-sa] [-sn] [-t | -u] [-w] [-y] [-z] [-zs #] [romname]
+\fBzsnes\fP [-?] [-0] [-1 #] [-2 #] [-7] [-8] [-9] [-a] [-cb] [-cc] [-cs] [-dd] [-f #] [-h | -l] [-i] [-j] [-k #] [-m] [-n] [-om] [-p #] [-r #] [-s] [-sa] [-sn] [-t | -u] [-w] [-y] [-z] [-zs #] <romname>
 
-.SH DESCRIPTION
-.B ZSNES
-is an emulator for the Super Nintendo video game console. It is written in a combination of x86 assembler, C and C++ and is currently available on Windows, DOS and x86 Linux.
-
-Visit http://www.zsnes.com/ for the latest updates.
-
-.SH OPTIONS
-Note: many of these options can also be set in the GUI
+.SS
 .TP
-.B -?
+\fB-?\fP
 Display command-line help
 .TP
-.B -0
+\fB-0\fP
 Disable color 0 modification in 256 (8-bit) color modes
 .TP
-.B -1 # / -2 #
-Set controls for Player 1/2.
-.nf
-\fI0\fP = None       \fI1\fP = Keyboard       \fI2\fP = Joystick       \fI3\fP = Gamepad 
-\fI4\fP = 4 Button   \fI5\fP = 6 Button       \fI6\fP = Sidewinder
-.fi
+\fB-1 # / -2 #\fP
+Set controls for player #1/#2:
+0 = None; 1 = Keyboard; 2 = Joystick; 3 = Gamepad; 4 = 4-Button; 5 = 6-Button; 6 = Sidewinder
 .TP
-.B -7
+\fB-7\fP
 Disable SPC700 speedhack
 .TP
-.B -8
+\fB-8\fP
 Force 8-bit sound
 .TP
-.B -9
+\fB-9\fP
 Off by 1 line fix
 .TP
-.B -a
-Enable automatic frame rate
+\fB-a\fP
+Turn on auto frame skip
 .TP
-.B -cb
+\fB-c\fP
+Scale to fit screen
+.TP
+\fB-cb\fP
 Remove Background Color in 256 (8-bit) color modes
 .TP
-.B -cc
-Enable small screen (when available)
+\fB-cc\fP
+Small screen mode (if available)
 .TP
-.B -cs
-Enable full/wide screen (when available)
+\fB-cs\fP
+Full/Wide screen mode (if available)
 .TP
-.B -dd
+\fB-dd\fP
 Disable sound DSP emulation
 .TP
-.B -f #
+\fB-f #\fP
 Enable frame skipping, where \fB#\fP is a numerical value from \fI0-9\fP
 .TP
-.B -h
+\fB-h\fP
 Force HiROM. Cannot be used with \fB-l\fP
 .TP
-.B -i
+\fB-i\fP
 Uninterleave ROM image
 .TP
-.B -j
+\fB-j\fP
 Disable Mouse (Automatically turns off right mouse click)
 .TP
-.B -k #
+\fB-k #\fP
 Set volume level, where \fB#\fP is a numerical value from \fI1-100\fP
 .TP
-.B -l
+\fB-l\fP
 Force LoROM. Cannot be used with \fB-h\fP
 .TP
-.B -m
+\fB-m\fP
 Disable GUI
 .TP
-.B -n
-Enable full scanlines (when available)
+\fB-n\fP
+Enable scanlines
 .TP
-.B -om
-Enable MMX support (when available)
+\fB-om\fP
+Enable MMX copy
 .TP
-.B -p #
-Change the clock speed of the main CPU, where
-.B #
-is a numerical value from \fI50-120\fP. 100 is the normal speed. Changing this value may have adverse side effects.
+\fB-p #\fP
+Percentage of instructions to execute, where \fB#\fP is a numerical value from \fI50-150\fP
 .TP
-.B -r #
-Set audio sampling rate, where
-.B #
-is one of:
-.nf
-\fI0\fP = 8 KHz       \fI1\fP = 11.025 KHz       \fI2\fP = 22.05 KHz
-\fI3\fP = 44.1 KHz    \fI4\fP = 16 KHz           \fI5\fP = 32 KHz
-.fi
+\fB-r #\fP
+Set audio sampling rate, where \fB#\fP is one of:
+
+\fI0\fP = 8 kHz        \fI1\fP = 11.025 kHz  \fI2\fP = 22.05 kHz     \fI3\fP = 44.1 kHz     \fI4\fP = 16 kHz        \fI5\fP = 32 kHz
 .TP
-.B -s
-Enable SPC700/DSP emulation (Sound)
+\fB-s\fP
+Enable SPC700/DSP emulation (i.e., sound)
 .TP
-.B -sa
-Show files with all extensions (*.*) in GUI. Default is to only show files with one of the extensions listed in
-.B romname
+\fB-sa\fP
+Show files with all extensions (*.*) in GUI. Default is to only show files with one of the extensions listed in \fBromname\fP
 .TP
-.B -sn
+\fB-sn\fP
 Enable snowy GUI background
 .TP
-.B -t
+\fB-t\fP
 Force NTSC (60 Hz) timing. Cannot be used with \fB-u\fP
 .TP
-.B -u
+\fB-u\fP
 Force PAL (50 Hz) timing. Cannot be used with \fB-t\fP
 .TP
-.B -w
-Enable VSync (disables triple buffering)
+\fB-w\fP
+Enable VSync
 .TP
-.B -y
-Enable EAGLE (640x480x8B only) or Interpolation (640x480x16B only)
+\fB-y\fP
+Enable EAGLE (640x480x256 <8-bit> only) or Interpolation (640x480x65536 <16-bit> only)
 .TP
-.B -z
-Disable stereo sound. Stereo sound is enabled by default.
+\fB-z\fP
+Disable stereo sound
 .TP
-.B -zs #
-Auto load specified save state on startup
+\fB-zs #\fP
+Load save state on startup
 .TP
-.B romname
-Name of ROM to auto-load. 
-.B romname
-is of the format \fBfilename.ext\fP, where
-.B .ext
-is one of
-.I .SMC, .SFC, .SWC, .FIG, .058, .078, .1, .A, .USA, .JAP,
-or \fI.GD3\fP.  You can also load ROMs which have been compressed with Zip, GZip or JMA.
+\fBromname\fP
+Name of ROM to auto-load. \fBromname\fP is of the format \fBfilename.ext\fP, where \fB.ext\fP is one of \fI.SMC, .SFC, .SWC, .FIG, .058, .078, .1, .A, .USA,\fP or \fB.JAP\fP  To save space, you can zip the ROM files and zSNES should still be able to load them.
 
+.SH DESCRIPTION
+\fBzSNES\fP is an emulator for the Super Nintendo video game console, written mostly in assembler by zsKnight and _Demo_ (who also works on ePSXe), with Pharos and Teuf as assistant coders. Originally a DOS program, it has now been ported to Windows and Linux. All core udpates will be done simultaneously (i.e. for DOS, Windows, and Linux versions).
 
+Visit http://www.zsnes.com for the latest updates.
+
+This manual page documents version 1.40 of zsnes, with a few updates for 1.42, but is not a reference for zsnes 1.42.
+
 .SH "FILES AND DIRECTORIES"
 .TP
 zsnes
 The main program
 .TP
 ~/.zsnes
-This is the directory where ZSNES configuration files are stored. By default, it is also the directory where save states, screenshots, SPC dumps and ZMV movie files are stored, but you can change this behaviour by editing zsnesl.cfg. This directory is created by ZSNES.
+The directory where the save states, screenshots, and movie files are stored. This directory is created by zsnes
 .TP
 ~/.zsnes/zsnesl.cfg
-This file is where configuration settings for ZSNES are stored. It is in a human-readable format and can be edited by hand. It is created by ZSNES.
+Where configuration settings are stored. This file is created by zsnes.
 .TP
 ~/.zsnes/.zguicfgl.dat
-This file is where settings for the GUI are saved. It is created by ZSNES.
+The dat file for the GUI. This file is created by zsnes. \fBIf for some reason you update your version of zsnes and it keeps crashing before you can do anything, you should try deleting this file and have zsnes create a new one.\fP
 
-.B If for some reason you update your version of ZSNES and it keeps crashing before you can do anything, you should try deleting the two above files and let ZSNES create new ones.
 
+.SH "GRAPHICS PACKS"
+The graphics packs for games such as Star Ocean, Street Fighter Alpha, and others must be stored in the ~/.zsnes directory. For example, if you have Star Ocean, you will need to place the graphics pack in ~/.zsnes/SOCNSDD1 and the files within that directory must all be lowercase. (This will remain this way until the authors decide to make changes in the init.asm file.) As of version 1.337, the directory name of the graphics packs must be capitalized and all files within that directory are in lowercase.
 
+
+.SH "SEE ALSO"
+\fIzsnesgui(1)\fP and \fIzsnesfaq(1)\fP, neither of which happens to be written yet. (Hey, this is a work in progress.)
+
+
 .SH "SPECIAL CHIP GAMES"
-Many games on the Super Nintendo used custom CPUs in addition to the main SNES CPU. Some of these CPUs have not been fully reverse-engineered, and games that use them may either not work or may experience strange bugs.
+Many games that use special chips do not work, including but not limited to the following:
 
-You can find more information about many of these special CPUs at http://users.tpg.com.au/trauma/dsp/ and http://nsrt.edgeemu.com/INFO/chipinfo.htm .
 
+SuperFX:
 
-.SH "GRAPHICS PACKS"
-Some games based on the SPC7110 chip, such as Far East of Eden Zero, can use special graphics packs to work around unemulated portions of their custom CPU. These graphics packs must be stored in individual directories within the save directory, which is specified in zsnesl.cfg. By default, this is ~/.zsnes. The directory name of each graphics pack must be capitalized and all files within that directory must be in lowercase.
+- Dirt Trax FX
 
-The proper directory name for a given graphics pack can be found in the readme file included with that pack.
+- Winter Gold
 
-NOTE: As of version 1.40, games which use the S-DD1 chip (Star Ocean, Street Fighter Alpha 2) do not require graphics packs.
 
-.SH "REPORTING BUGS"
-You can report bugs using the bug tracker at the ZSNES SourceForge page, located at http://sourceforge.net/projects/zsnes/.
+DSP-1 (and variants):
 
-.SH AUTHORS
-Please see the authors.txt file included in the ZSNES archive.
+- Pilotwings
 
-.SH "COPYRIGHT NOTICE"
-Copyright \(co 1997-2004 the ZSNES team
+- Top Gear 3000
 
-Permission is granted to copy and distribute this  manual  under the terms of the GNU Free Documentation License.
\ No newline at end of file
+
+SDD-1:
+
+- Street Fighter Alpha 2 now works with the DeJap graphics pack.
+
+- Star Ocean now works with the DeJap graphics pack.
+
+
+SPC7110:
+
+- Far East of Eden Zero works with the DeJap graphics pack.
+
+
+SA-1:
+
+- Super Mario RPG: Legend of the Seven Stars
